bitnami/dataplatform-bp1/Chart.yaml

```
@@ -47,4 +52,8 @@ sources:
   - https://github.com/bitnami/bitnami-docker-solr
   - https://zookeeper.apache.org/
   - https://github.com/bitnami/bitnami-docker-zookeeper
-version: 0.1.0
+  - https://github.com/bitnami/bitnami-docker-wavefront-kubernetes-collector
+  - https://github.com/bitnami/bitnami-docker-wavefront-proxy
+  - https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes
+  - https://github.com/wavefrontHQ/wavefront-proxy
+version: 1.0.0
```

Is backward compatibility guaranteed? If the changes in this PR are not going to break the upgrades from the previous version, we only need to bump the chart version in a minor instead of a major
```suggestion
version: 0.2.0
```

---

Per the above explanation, this field should be updated using semver, in this case as it is a feature addition and not a backward incompatibility we should bump the `version` in a minor.

---

Updated the version as per the comments. Chart version now is 0.2.0 and appVersion is 1.0.0.